usage-examples/test-execution.md

```diff
@@ -52,13 +52,5 @@ To signal that a test case is about to be executed an [EiffelTestCaseTriggeredEv
 A [EiffelConfidenceLevelModifiedEvent](../eiffel-vocabulary/EiffelConfidenceLevelModifiedEvent.md) is sent to declare what level of confidence was reached for the given item under test. It COULD link to the executed test suite to track what tests where performed to achieve the given confidence level.
 
 ## Test Activity Execution Implementation
+The [Eiffel Sepia architecture description](https://eiffel-community.github.io/eiffel-sepia/index.html) provides [a possible sequence diagram of roles interacting](https://eiffel-community.github.io/eiffel-sepia/test-orchestration.html) to realize the execution and reporting of test cases.
 To realize the event graph in this example, a sequence like the following is needed:
-![alt text](./test-execution-sequence.png "Sequence of Calls and Events of Test Execution Example")
```

Comment:
I'd prefer that the sequence diagram was kept in this PR. I agree that the Sepia link could possibly replace it but that is a different story and more stuff on this page needs to be updated or migrated to Sepia before that can be done. The third section in the Introduction on this page depends on the sequence diagram being here for example.